apps/onboarding-functions/src/main/resources/application.properties

```diff
@@ -16,7 +16,7 @@ quarkus.index-dependency.onboarding.artifact-id=onboarding-sdk-common
 onboarding-functions.retry.max-attempts = ${RETRY_MAX_ATTEMPTS:5}
 onboarding-functions.retry.first-retry-interval = ${FIRST_RETRY_INTERVAL:5}
 onboarding-functions.retry.backoff-coefficient = ${BACKOFF_COEFFICIENT:5}
-
+onboarding-function.force-institution-persist= ${FORCE_INSTITUTION_PERSIST:false}
```

Comment:
you should added `FORCE_INSTITUTION_PERSIST=true` under infra/container_apps for pnpg environment

Reply:
should we set this to true straight away?

Reply:
no, default false is okay!